doc/bus.rst

```diff
@@ -37,10 +39,24 @@ Receiving
 '''''''''
 
 Reading from the bus is achieved by either calling the :meth:`~can.BusABC.recv` method or
-by directly iterating over the bus::
+by directly iterating over the bus:
```

Comment:
This is not a mistake, see [here](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#indented-literal-blocks).